# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/20 03:17:31+01:00 matthias.andree@gmx.de 
#   MCONFIG.in:
#     Rearrange -I options to give local includes precedence over
#     /usr/local/include, to unbreak the build when an older
#     e2fsprogs version (with includes) is installed under
#     /usr/local (for instance, on FreeBSD).
# 
# MCONFIG.in
#   2005/01/20 03:16:52+01:00 matthias.andree@gmx.de +2 -3
#   Rearrange -I options to give local includes precedence over
#   /usr/local/include, to unbreak the build when an older
#   e2fsprogs version (with includes) is installed under
#   /usr/local (for instance, on FreeBSD).
# 
diff -Nru a/MCONFIG.in b/MCONFIG.in
--- a/MCONFIG.in	2005-01-20 03:43:02 +01:00
+++ b/MCONFIG.in	2005-01-20 03:43:02 +01:00
@@ -34,11 +34,10 @@
 BUILD_CC = @BUILD_CC@
 DEFS = -DLOCALEDIR=\"$(localedir)\" @DEFS@
 CFLAGS = @CFLAGS@
-CPPFLAGS = @CPPFLAGS@
+CPPFLAGS =  -I$(top_builddir)/lib -I$(top_srcdir)/lib @CPPFLAGS@
 INTL_FLAGS = @INTL_FLAGS@
 ALL_CFLAGS = $(CPPFLAGS) $(DEFS) $(USE_WFLAGS) $(CFLAGS) $(XTRA_CFLAGS) \
-	$(INTL_FLAGS) -I$(top_builddir)/lib -I$(top_srcdir)/lib \
-	$(LINUX_INCLUDE) 
+	$(INTL_FLAGS) $(LINUX_INCLUDE) 
 LDFLAGS = @LDFLAGS@
 ALL_LDFLAGS = $(LDFLAGS)
 RM = @RM@
